pkg/databind/pkg/databind/replacer.go

```diff
@@ -151,14 +151,17 @@ func replaceFields(values []data.Map, val reflect.Value, rc replaceConfig, match
 		nStr := replaceBytes(values, []byte(val.String()), rc, matches)
 		return reflect.ValueOf(string(nStr))
 	case reflect.Map:
```

Comment:
should we add something similar also for slices?

Reply:
Make sense, added in https://github.com/newrelic/infrastructure-agent/pull/1292/commits/d2fc3e3f5c5b5294235ac571c2c4a6c2ad074974